File: src/main/java/com/distribution/modules/sys/entity/District.java
package com.distribution.modules.sys.entity;

import lombok.Data;

/**
 * @author ChunLiang Hu
 * @Company
 * @Project distribution
 * @Package com.distribution.modules.sys.entity
 * @Description TODO(描述)
 * @create 2018/6/20-21:59
 */
@Data
public class District {
    private Integer id;
    private String name;
    private District parent;
    private String code;
    private Integer order;
}

File: src/main/kotlin/com/distribution/modules/sys/entity/District.kt
package com.distribution.modules.sys.entity

import java.io.Serializable


/**
 * @author ChunLiang Hu
 * @Company
 * @Project distribution
 * @Package com.distribution.modules.sys.entity
 
 * @create 2018/6/20-21:59
 */

data class District(
        private val serialVersionUID: Long = 1L,
        var id: Int? = null,
        var name: String? = null,
        var parent: District? = null,
        var code: String? = null,
        var order: Int? = null
) : Serializable
